scalafix-tests/unit/src/test/scala/scalafix/tests/cli/CliSemanticSuite.scala

```diff
@@ -170,7 +170,7 @@ class CliSemanticSuite extends BaseCliSuite {
   checkSemantic(
     name = "-P:semanticdb:targetroot",
     args = {
-      val (targetroot :: classDir :: Nil, jars) =
+      val (_ :: targetroot :: Nil, jars) =
```

Comment:
not new to this PR, but the fact that order matters here makes the test a bit fragile - I think it's acceptable as we don't put any ordering constraint for (external) testkit client, this is just a shortcut for fixturing.